pylutron/__init__.py

```diff
@@ -18,6 +18,17 @@
 
 _LOGGER = logging.getLogger(__name__)
 
+# We brute force exception handling in a number of areas to ensure
+# connections can be recovered
+_KNOWN_EXCEPTIONS = (
```

Comment:
Maybe _EXPECTED_NETWORK_EXCEPTIONS or something like that? KNOWN is a bit ambiguous, otherwise I like it.